eclipse-theia/theia-blueprint
# Update to 1.37.0

#### What it does
Updates Theia to 1.37.0 and upgrades dependencies. 
Applies the nexessary fixes required because of the `@electron/remote` removal (closes #258 )

#### How to test
Build Theia Blueprint and check that the build and the application is still working.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
